python/src/aiconfig/editor/client/src/themes/GradioTheme.ts

```diff
@@ -87,17 +87,6 @@ export const GRADIO_THEME: MantineThemeOverride = {
             color: "#374151",
           },
 
-        /*
-         * Specifically set the .runPromptButton to have the desired styles specified in our custom gradio theme
-         */
-        ".mantine-Button-root.mantine-UnstyledButton-root.runPromptButton": {
```

Comment:
Shouldn't be needed if the runPromptButton class specificity is high enough now below

Reply:
Sorry, just to avoid confusion, when you say "high enough" you don't mean "higher up the component stack towrads ancestors", you mean "closer enough to the child component so that it overwrites the parent-defined style" right?

Reply:
Actually, I mean the specificity itself has the highest relevance for the matching elements based on the combination of selectors (descendant of div.editorBackground AND runPromptButton class) -- see https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity for more info